torch_spyre/csrc/spyre_tensor_impl.cpp

```diff
@@ -40,6 +40,19 @@ void SpyreTensorLayout::init(std::vector<int64_t> host_size,
                              c10::ScalarType dtype,
                              std::vector<int32_t> dim_order,
                              StickFormat format) {
+  if (host_size.size() == 0) {
```

Comment:
@avery-blanchard @thoangtrvn -- When we send a constant to the device as a tensor, does the DMA broadcast it  (so for example for a float16, we have a dense tensor of size 64 with the same value in all 64 elements) or do we end up with a sparse tensor (tensor of size 1 with sparse stick (1 value, 63 garbage values).